src/content/docs/guides/platform-setup/configure-reverse-proxy.mdx

```diff
@@ -19,7 +17,15 @@ distinct entry points with 4 different URLs:
 4. **Platform API**: The URL the Tenzir Platform CLI connects to, e.g.,
    `api.platform.example`.
 
-Set the following environment variables to reach these 4 services:
+The typical setup for the Tenzir Platform uses a reverse proxy to terminate
+TLS connections and forward incoming traffic to the correct services. The choice
+of technology varies with your deployment scenario and ranges from an additional
+`traefik` container within the docker compose stack, over an nginx instance
```

Comment:
Better: use the product name here, i.e., **Traefik**, and link to the project site.